benchmark/weak_scaling_shallow_water_model.jl

```diff
@@ -0,0 +1,42 @@
+using JLD2
+using BenchmarkTools 
+using Benchmarks
+
+# Benchmark parameters
+
+ Nx = 4096
+Nyp = 256
```

Comment:
I called this `Nyp` because it's the number of `Ny` per core.  Not a great name but it does work

Reply:
I just added a comment for clarity.